api-portal/install-api-portal.hbs.md

```diff
@@ -70,3 +75,57 @@ To install API portal:
 
     Added installed package 'api-portal' in namespace 'tap-install'
     ```
+
+5. Verify the package installation by running:
+
+    ```console
+    tanzu package installed get api-portal -n tap-install
+    ```
+
+   Verify that `STATUS` is `Reconcile succeeded`:
+
+    ```console
+    kubectl get pods -n api-portal
+    ```
+
+## <a id='update-values'></a>Update the install values for api-portal package
+
+1. To overwrite the default values, create new values, or update the existing values, you will need an
+   `api-portal-values.yaml` file. If you do not already have an existing values file, you can extract the existing values using the command:
+
+    ```console
+    tanzu package installed get api-portal -n api-portal -f api-portal-values.yaml
```

Comment:
If we follow the same steps mentioned before this section, the namespace used for installing api-portal should be `tap-install`:
```suggestion
    tanzu package installed get api-portal -n tap-install -f api-portal-values.yaml
```